unitTests/ui/ComponentOptionsTest.ts

```diff
@@ -575,10 +574,29 @@ export function ComponentOptionsTest() {
       });
 
       describe('findParentScrolling', () => {
-        it('which finds a scrollable parent', () => {
+        it('which finds a scrollable parent with overflow scroll', () => {
+          scrollElem.style.overflowY = 'scroll';
+          const option = ComponentOptions.findParentScrolling(elem, doc);
+          expect(option).toBe(scrollElem);
+        });
+
+        it('which finds a scrollable parent with overflow auto', () => {
+          scrollElem.style.overflowY = 'auto';
           const option = ComponentOptions.findParentScrolling(elem, doc);
           expect(option).toBe(scrollElem);
         });
+
+        it("which won't detect overflow visible as scrollable", () => {
+          scrollElem.style.overflowY = 'visible';
+          const option = ComponentOptions.findParentScrolling(elem, doc);
+          expect(option).not.toBe(scrollElem);
+        });
+
+        it("which won't detect overflow inherit as scrollable", () => {
```

Comment:
The `inherit` value is interesting in the cases where the parent has the value set to `scroll` or `auto`. Could such cases happen?